packages/dashboard/src/pages/Settings/components/AppearanceSetting.tsx

```diff
@@ -0,0 +1,20 @@
+import { MenuItem } from '@material-ui/core'
+import { Services } from '../../../API'
+import { useAppearance } from '../../Personas/api'
+
+import SettingSelect from './SettingSelect'
+
+export default function AppearanceSetting() {
+    let mode = useAppearance()
+    const handleChange = (event: any) => {
+        Services.Settings.setTheme(event.target.value)
+    }
+
+    return (
+        <SettingSelect value={mode} onChange={handleChange}>
+            <MenuItem value="default">Follow system settings</MenuItem>
+            <MenuItem value="light">Light</MenuItem>
+            <MenuItem value="dark">Dark</MenuItem>
```

Comment:
Although it isn't likely to being changed, but I think it's better to use 
```
<MenuItem value={Appearance.default}>Light</MenuItem>
```

here.

`Appearance` is defined in the maskbook package. I think we can move it to the `theme` package.

Reply:
done